cmd/browser-peer-rpc_test.go

```diff
@@ -41,9 +41,8 @@ func (s *TestRPCBrowserPeerSuite) SetUpSuite(c *testing.T) {
 }
 
 // No longer used with gocheck, but used in explicit teardown code in
-// each test function. // Called implicitly by "gopkg.in/check.v1"
-// after all tests are run.
-func (s *TestRPCBrowserPeerSuite) TearDownSuite(c *testing.T) {
+// each test function. Called implicitly by after all tests are run.
```

Comment:
Maybe a typo. Comment needs to be clear.

Reply:
Sure